publiq-ui/pub-modal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <b-modal
+    cancel-variant="outline-secondary"
+    :title="title"
+    :ok-title="confirmTitle"
+    :cancel-title="cancelTitle"
+    title-class="title"
+    modal-class="modal"
+    content-class="content"
+    body-class="body"
+    :visible="visible"
+    :scrollable="false"
+    :hide-footer="isContentVariant"
+    :hide-header="isQuestionVariant"
```

Comment:
Should this actually be hidden? See the screenshots in my comment here https://github.com/cultuurnet/udb3-frontend/pull/46#issuecomment-688422917

The AngularJS example does have a header with a title.

Reply:
I looked at another example 
![Schermafbeelding 2020-09-09 om 09 39 36](https://user-images.githubusercontent.com/66943525/92569093-702db200-f280-11ea-90f3-c71797f932da.png)

I'll hide-header it as a prop we can pass to pub-modal

Reply:
Maybe we can show/hide the header depending on the value of the `title` prop instead? If it's empty hide it, if it has a value show it?

Reply:
Also a possibility.